### ISSUE TITLE
a cascade replication slave  did not sync data as expected by redisShake 

### ISSUE THREAD

**Issue by yotone** [aliyun/redis-shake#277]
## My preperation
There are three redis instance:redis:6379 redis:6380 redis:6381.Redis:6380 is the slave of redis:6379, and redis:6381 is a normal master-role instance.In addition my redis version is 5.0.7.
I want sync  redis:6380 db0 to redis:6381 db1 by redisShake，so I configed  target.db = 1,source.address and target.address and so on. 
```
source.address = 127.0.0.1:6380
target.address = 127.0.0.1:6381
target.db = 1

```
## My operation
Then I started redisShake like : 
``` 
 nohup  ./redis-shake.linux -conf redis-shake.conf -type sync &
 ```
The process didn't have any problem util full sync. But when an incremental sync started , I set a key in redis:6379 db0 .
```
127.0.0.1:6379> set water 123
```
Then I found the key was writen in redis:6381 db0 not as I expected in db1.

```

127.0.0.1:6381>  info keyspace
# Keyspace
db0:keys=1,expires=0,avg_ttl=0
db1:keys=1,expires=0,avg_ttl=0
127.0.0.1:6381> keys *
1) "water"

**Comment by vinllen** [comment]
The reason why the `target.db` isn't work is that in cascade replication, the proxy redis which also means the `redis:6380` won't replicate the `select` command to the downstream. But in master-slave scenario, the `select` command will be produced.
This bug has been fixed since v2.0.2, please checkout and retry. Let me know if you have any questions.

**Comment by yotone** [comment]
I have already made a test , and it worked. 
Thanks.
